module/verifier.go

```diff
@@ -4,11 +4,13 @@ import (
 	"github.com/onflow/flow-go/crypto"
 )
 
+// TODO : to delete in V2
```

Comment:
I think we still need the Verifier, the consensus follower is able to verify and sign a block in a contextless manner
```suggestion
```

Reply:
Yes, I meant we won't need that exact interface.